lib/src/rules/prefer_foreach.dart

```diff
@@ -128,8 +128,11 @@ class _Visitor extends SimpleAstVisitor {
   _Visitor(this.rule);
 
   @override
-  visitForEachStatement(ForEachStatement node) {
+  visitForStatement2(ForStatement2 node) {
+    final loopParts = node.forLoopParts;
+    if (loopParts is ForEachParts) {
     final visitor = new _PreferForEachVisitor(rule);
     node.accept(visitor);
```

Comment:
Maybe it's the display on GitHub, but it looks like the body isn't indented.

Reply:
Nope.  Good eye.  Caught also by travis:

https://travis-ci.org/dart-lang/linter/jobs/502837949#L1030

Reply:
Fixed, thanks!